Sources/FuntastyKit/Architecture/Coordinator.swift

```diff
@@ -32,8 +32,10 @@ public protocol DefaultCoordinator: Coordinator {
 }
 
 public protocol ShowCoordinator: DefaultCoordinator {
-    var sourceViewController: UIViewController { get }
+    /// sourceViewController should be used as 'weak var', otherwise memory leak will emerge
+    var sourceViewController: UIViewController? { get }
```

Comment:
Where is `weak` keyword? Shouldn't be here at the start of line?

Reply:
'weak' cannot be applied to a property declaration in a protocol

Reply:
@bezoadam very good england exhibition here. 👏 ...jinak rekl bych, ze sis nevsiml toho, ze to je protocol, takze tady to weak byt nemuze, ale aby ten `sourceViewController` mohl byt weak v konkretnim Coordinatoru, tak musi byt optional, protoze weak veci musi byt vzdycky optional.

Reply:
shit a ja si nevsiml, ze to je do FuntastyKitu, proto ta anglictina :loool: